### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:
Adjusts the ignore list to cover generated files, ie `...generated.go` and `....pb.go`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-launcher/virtwrap/manager_test.go [hunk 1]
```diff
@@ -1711,6 +1711,129 @@ var _ = Describe("Manager", func() {
 		})
 	})
 
+	Context("other unit tests", func() {
```
Comment (rmohr) [inline pkg/virt-launcher/virtwrap/manager_test.go:1714]: if you don't think it is worth a specific context, consider just removing the context.

Comment (dhiller) [reply pkg/virt-launcher/virtwrap/manager_test.go]: Thanks! Done, PTAL.

File: pkg/virt-launcher/virtwrap/manager_test.go [hunk 2]
```diff
@@ -1737,6 +1737,121 @@ var _ = Describe("Manager", func() {
 		})
 	})
 
+	It("executes hotPlugHostDevices", func() {
+		os.Setenv("KUBEVIRT_RESOURCE_NAME_test1", "127.0.0.1")
+		os.Setenv("PCIDEVICE_127_0_0_1", "05EA:Fc:1d.6")
```
Comment (rmohr) [inline pkg/virt-launcher/virtwrap/manager_test.go:1742]: 👍 Let's fix this in a follow-up in the code itself. Env vars should be read in the outmost app layer and then passed via variables or struct members.

**Pull request merged**